admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatter.vue

```diff
@@ -28,8 +28,9 @@
       <language-list
         v-if="languagesCount"
         :languages="languages"
+        :translations="translations"
```

Comment:
You're not supposed to inject the translations, Vue integrate a mixin to deal with translations thanks to the `this.$t()` function